examples/Utils/b3Clock.cpp

```diff
@@ -112,16 +112,14 @@ void b3Clock::reset()
 unsigned long int b3Clock::getTimeMilliseconds()
 {
 #ifdef B3_USE_WINDOWS_TIMERS
-	LARGE_INTEGER currentTime;
-	QueryPerformanceCounter(&currentTime);
-	LONGLONG elapsedTime = currentTime.QuadPart - 
-		m_data->mStartTime.QuadPart;
-		// Compute the number of millisecond ticks elapsed.
-	unsigned long msecTicks = (unsigned long)(1000 * elapsedTime / 
-		m_data->mClockFrequency.QuadPart);
-	
-
-		return msecTicks;
+		LARGE_INTEGER currentTime, elapsedTime;
```

Comment:
Is this just a formatting/layout fix? Please undo (it makes reviewing very hard to mix layout changes with actual changes)